presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -666,11 +706,11 @@ public void testPushdownComposition()
         // Tests composing two pushdowns each with a range filter and filter function.
         assertQuery(
                 "WITH data AS (" +
-                    "    SELECT l.suppkey, l.linenumber, l.shipmode, MAX(o.orderdate)" +
-                    "    FROM lineitem l,  orders o WHERE" +
-                    "        o.orderkey = l.orderkey AND linenumber IN (2, 3, 4, 6) AND shipmode LIKE '%AIR%'" +
-                    "        GROUP BY l.suppkey, l.linenumber, l.shipmode)" +
-                    "SELECT COUNT(*) FROM data WHERE suppkey BETWEEN 10 AND 30 AND shipmode LIKE '%REG%'");
+                        "    SELECT l.suppkey, l.linenumber, l.shipmode, MAX(o.orderdate)" +
```

Comment:
there is a bad formatting on the exiting file. When auto format is used, this gets changed. This should fix this once for all, but sorry for the unrelated change here